frontend/src/pages/spotnet/dashboard/add_deposit.jsx

```diff
@@ -0,0 +1,55 @@
+
+import React,{useState} from 'react';
+import Card from 'components/Card/Card';
+import './add_deposit.css';
+import TokenSelectorDeposit from 'components/TokenSelector/TokenSelector';
+import { ReactComponent as HealthIcon } from 'assets/icons/health.svg';
+import { ReactComponent as EthIcon } from 'assets/icons/ethereum.svg';
+export default function Add_Deposit() {
+   const [amount, setAmount] = useState('0');
+  const handleAmountChange = (e) => {
+    const value = e.target.value;
+    const regex = /^\d*\.?\d*$/;
+    if (regex.test(value)) {
+      setAmount(value);
+    }
+  };
+ 
+  return (
+    <div className="deposit-wrapper">
+      <div className="deposit-container">
+        <h1 className="deposit-title1">zkLend Deposit</h1>
+        <div className="main-container-deposit">
+          <div className="top-cards-deposit">
+            <Card label="Health Factor" value="1.47570678" icon={<HealthIcon className="icon" />} />
+            <Card label="Borrow Balance" value="$-55.832665" icon={<EthIcon className="icon" />} />
```

Comment:
cards do not look like in figma, everything is different, pleae make changes to css and verify it looks just like in figma on both mobile and desktop